lib/stagehand/staging/synchronizer.rb

```diff
@@ -1,6 +1,8 @@
 module Stagehand
   module Staging
     module Synchronizer
+      include Stagehand::Database
```

Comment:
You'll notice that you're able to call the methods directly on the module _or_ include them in another object. This is because of `extend self`. See http://idiosyncratic-ruby.com/8-self-improvement.html. I could have used `module_function` as is suggested, but I didn't know about it until today. Thanks rubocop.